cluster-autoscaler/Dockerfile

```diff
@@ -30,7 +30,15 @@ RUN cd autoscaler/cluster-autoscaler \
 # This is adapted from CA's Dockerfile.amd64, here:
 # https://github.com/kubernetes/autoscaler/blob/cluster-autoscaler-1.24.1/cluster-autoscaler/Dockerfile.amd64
 
-FROM gcr.io/distroless/static:nonroot-amd64
+
+# NB: two build stages, one for each architecture, because I wasn't able to use variable substitution in FROM statements
```

Comment:
Hmm, I wonder if we do something different here, usually it works like this:

```dockerfile
ARG ARCH
FROM gcr.io/distroless/static:nonroot-${ARCH}
```

Reply:
TIL, that Dockerfile args can have different scopes. Thanks for pointing out.

>Build arguments declared in the global scope of a Dockerfile aren't automatically inherited into the build stages. >They're only accessible in the global scope.

https://docs.docker.com/build/building/variables/#scoping